[2024-02-01 13:29:01,625] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.create_ddl_postgres manual__2024-02-01T13:29:01.201459+00:00 [queued]>
[2024-02-01 13:29:01,633] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.create_ddl_postgres manual__2024-02-01T13:29:01.201459+00:00 [queued]>
[2024-02-01 13:29:01,634] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 13:29:01,634] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-01 13:29:01,634] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 13:29:01,644] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_ddl_postgres> on 2024-02-01 13:29:01.201459+00:00
[2024-02-01 13:29:01,648] {standard_task_runner.py:52} INFO - Started process 3338 to run task
[2024-02-01 13:29:01,652] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'create_ddl_postgres', 'manual__2024-02-01T13:29:01.201459+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpckxiaomq', '--error-file', '/tmp/tmplxt530nt']
[2024-02-01 13:29:01,654] {standard_task_runner.py:80} INFO - Job 258: Subtask create_ddl_postgres
[2024-02-01 13:29:01,714] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.create_ddl_postgres manual__2024-02-01T13:29:01.201459+00:00 [running]> on host 0cb2a3b23a40
[2024-02-01 13:29:01,782] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=create_ddl_postgres
AIRFLOW_CTX_EXECUTION_DATE=2024-02-01T13:29:01.201459+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-01T13:29:01.201459+00:00
[2024-02-01 13:29:04,823] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-01 13:29:07,918] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-01 13:29:07,956] {logging_mixin.py:115} INFO - success migrate table
[2024-02-01 13:29:07,956] {python.py:173} INFO - Done. Returned value was: None
[2024-02-01 13:29:07,967] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=create_ddl_postgres, execution_date=20240201T132901, start_date=20240201T132901, end_date=20240201T132907
[2024-02-01 13:29:08,003] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-01 13:29:08,022] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
